packages/x-charts-pro/src/context/ZoomProvider/useSetupZoom.ts

```diff
@@ -89,6 +90,13 @@ export const useSetupZoom = () => {
       }
 
       event.preventDefault();
+      if (interactionTimeoutRef.current) {
+        clearTimeout(interactionTimeoutRef.current);
+      }
+      setIsInteracting(true);
+      interactionTimeoutRef.current = window.setTimeout(() => {
+        setIsInteracting(false);
+      }, 166);
```

Comment:
Is 166 a magic number? 🧙‍♂️

More seriously, what's the reason for this timeout?

Reply:
It came from the `mui/utils/debounce`. It is 10 frames at 60hz apparently. We can probably remove it or make it smaller, since the timeout will run in a new cycle anyways. We just can't have `setZoomData(newZoomData);` and `setIsInteracting(false);` together because react bundles them up

Reply:
Also, this is a wheel event, so there is no "wheelup" for us to turn the toggle off. So we pretty much reset the timer if we have another wheel event and after a few ms we reset it to false again.

Reply:
Maybe a comment

```js
// Debounce transition to `isInteractive=false`.
// Useful for wheel events which doesn't have an end event
```